src/charm.py

```diff
@@ -729,6 +733,12 @@ def _on_leader_elected_replica_data_handler(self, event):
         Only the leader can update the data shared with all replicas. Leader should check if
         the data exist when leadership is established, generate required data and set it in
         the peer relation if not.
+
+        Args:
+            event: required by ops framework, not used.
+
+        Returns:
```

Comment:
The returns is usually optional if there is nothing being returned

Reply:
I will remove them in the future, thanks!